### DISCUSSION TITLE
LFO key sync

### DISCUSSION THREAD

**Discussion by drmanz** [bangcorrupt/hacktribe#324] [Q&A] [answered]
Is possible disable the key sync of the bpm sync lfo?

**Comment by drmanz** [comment]
or in any case a choice between the 2 types

**Comment by bangcorrupt** [comment, marked as answer]
Yes, I will add this at some point, I have a few modulation hacks planned.  There are sine and unipolar square LFOs, resonance, attack and decay as modulation destinations, key retrigger and BPM sync for all the LFOs.  I haven't looked at the envelopes yet.

**Reply by ptowns** [reply]
Lfo to decay would be awesome! Also more then 1 slot would help?

**Reply by bangcorrupt** [reply]
I want to try assigning the existing modulations to different parts, in theory this shouldn't have much effect on processing power.
